books/free-programming-books-ar.md

```diff
@@ -34,15 +36,15 @@
 * [مدخل إلى الذكاء الاصطناعي وتعلم الآلة](https://academy.hsoub.com/files/17-%D9%85%D8%AF%D8%AE%D9%84-%D8%A5%D9%84%D9%89-%D8%A7%D9%84%D8%B0%D9%83%D8%A7%D8%A1-%D8%A7%D9%84%D8%A7%D8%B5%D8%B7%D9%86%D8%A7%D8%B9%D9%8A-%D9%88%D8%AA%D8%B9%D9%84%D9%85-%D8%A7%D9%84%D8%A2%D9%84%D8%A9/) - Mohamed Lahlah (PDF)
 
 
-### <a name="csharp"></a>C\#
+### <a name="csharp"></a>C#&lrm;
```

Comment:
I was rattled about removing the `\#` escape character but I will also say that after testing, it is just equal valid. In other words, good enhacement to bypass [markdownlint MD003 rule](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md003) using other way.

Reply:
Why don't we just use span with id ? to avoid `\#`, if you want.

```
### <span id="csharp">C#&lrm;</span>
```

### <span id="csharp">C#&lrm;</span>